projects/fsastorefrontlib/src/core/my-account/facade/quote.service.ts

```diff
@@ -72,33 +70,34 @@ export class QuoteService {
   retrieveQuote(quote: any) {
     this.userIdService
       .getUserId()
-      .pipe(take(1))
       .subscribe(occUserId => {
         if (occUserId) {
           this.cartService.loadCart(quote.cartCode, occUserId);
         }
       })
       .unsubscribe();
 
-    this.cartService
-      .getActive()
-      .pipe(
-        filter(cart => cart.code === quote.cartCode),
-        take(1)
-      )
-      .subscribe((cart: FSCart) => {
-        if (cart && cart.entries && cart.entries.length > 0) {
-          const orderEntry: OrderEntry = cart.entries[0];
-          const product: FSProduct = orderEntry.product;
-
-          this.loadPersonalDetailsForm(orderEntry);
-          this.loadChooseCoverForm(
-            cart.insuranceQuote,
-            product.defaultCategory.code
-          );
-        }
-      })
-      .unsubscribe();
+    this.subscription.add(
```

Comment:
Adding subscription like this in service doesn't help since service doesn't get destroyed, it's always active. If possible this should be transformed here and consumed in the components.